src/v3/test/integration/__snapshots__/authenticator-email-verification-data.test.tsx.snap

```diff
@@ -111,12 +112,9 @@ exports[`authenticator-email-verification-data should render form 1`] = `
                                 User
                               </title>
                             </svg>
-                          </span>
+                          </div>
                           <span
-                            aria-label="tester@okta1.com"
-                            class="identifier no-translate MuiBox-root emotion-17"
-                            data-se="identifier"
-                            title="tester@okta1.com"
```

Comment:
Are these attributes ok to remove? `data-se`, `aria-label`, `title` seem important

Reply:
I believe there are selenium tests within the monolith that uses this data-se attribute (we should probably be periodically running the downstreams against this topic branch to avoid a delay once this is completed. 

Also, re: the title, this was used because if the user ID is extremely long, we have css that truncates it and adds an ellipsis, the title was for visual users that allows them to hover the container to see the full user ID. For that reason I think we may need to keep this unless design says otherwise (it was originally a directive from UX to add this).

Reply:
As for `title` attribute, I can add it to the outer `Box` instead of `Tag` (which doesn't support it). Then it would show tooltip as before.

As for `data-se="identifier"`, it's applied to outer `div` (line 92)
I hope monolith selenium tests don't use `.identifier` class.
And I hope it apply `innerText` instead of `textContent` against element with `data-se="identifier"`. Otherwise we can remove `titleAccess="User"` from `<UserIcon>` to delete `<title>User</title>` inside icon